go.mod

```diff
@@ -49,5 +136,5 @@ replace (
 	github.com/tinkerbell/cluster-api-provider-tinkerbell => github.com/pokearu/cluster-api-provider-tinkerbell v0.0.0-20220128001529-79d851d0861f
 
 	// need the modifications eksa made to the capi api structs
-	sigs.k8s.io/cluster-api => github.com/mrajashree/cluster-api v1.0.3-0.20220301005127-382d70d4a76f
+	sigs.k8s.io/cluster-api => github.com/a-cool-train/cluster-api v1.0.3-custom.0.20220321135508-ebcbebe2dbfe
```

Comment:
Is this necessary?

Reply:
Yes momentarily. Cluster-api had an incompatible logr version, I fixed it in my fork and it might take sometime to get the necessary approvals. Once I get the approvals, I'll be reverting this change.

Reply:
Wait so did we fix this? If not, we should actually change it in Rajashree's fork itself because we use that as the one that includes all of our patches from main.